modules/root/root_v2_api.rb

```diff
@@ -25,6 +25,7 @@ class Proxy::RootV2Api < Sinatra::Base
   end
 
   def process_capabilities(state, capabilities)
+    return [] if capabilities.nil?
```

Comment:
Thanks to @adamruzicka and smart_proxy_salt we figured out how this can happen: if the plugin isn't properly initialized, `capabilities` can be nil. That's why you can see it in tests.

Now I wonder if this should be looking at `state` as well. If state is `:running` and `capabilities` is nil then it is a bug and we shouldn't accept that.